packages/devtools/lib/src/ui/drag_scroll.dart

```diff
@@ -60,4 +67,57 @@ class DragScroll {
       }
     });
   }
+
+  void _handleTouchDrags(Element dragged) {
+    num lastX;
+    num lastY;
+    bool touched = false;
+
+    dragged.onTouchStart.listen((event) {
+      final TouchEvent t = event;
+      // If there are multiple touches, always use the first.
+      final Touch touch = t.touches.first;
+
+      touched = true;
+      wasDraggedByTouch = false;
+
+      lastX = touch.client.x;
+      lastY = touch.client.y;
+
+      t.preventDefault();
+    });
+
+    window.onTouchEnd.listen((event) {
```

Comment:
these are on the window. Can they be added to dragged instead or at least added only when a touch starts and removed when it ends?

Reply:
These are on window so that the drag can extend beyond the `dragged` element as long as it started within it. We do the same thing for mouse scrolls. Otherwise where your drag ends (mouseUp or touchEnd) is constrained to the content size of `dragged`.